eq-author/src/components/EditorLayout/Header/index.js

```diff
@@ -97,24 +97,32 @@ export const UnconnectedHeader = props => {
         </RouteButton>
       );
     }
+
     if (publishStatus === AWAITING_APPROVAL && !me.admin) {
       return null;
     }
-    return (
-      <RouteButton
-        variant="tertiary-light"
-        to={buildPublishPath(match.params)}
-        small
-        disabled={
-          questionnaire.totalErrorCount > 0 ||
-          title === "Publish" ||
-          publishStatus === PUBLISHED
-        }
-        data-test="btn-publish"
-      >
-        <IconText icon={publishIcon}>Publish</IconText>
-      </RouteButton>
-    );
+
+    if (
+      me.admin ||
+      questionnaire.createdBy.id === me.id ||
+      questionnaire.editors.some(el => el.id === me.id)
+    ) {
+      return (
```

Comment:
Judging by the trello card I think the button is supposed to be disabled, not hidden when you can't publish.